### ISSUE TITLE
Crash on opening Minecraft 1.7.2 + Forge

### ISSUE THREAD

**Issue by keli5** [MultiMC/MultiMC5#3160]
System Information
-----------------------------
MultiMC version: 0.6.11-1344 on lin64

Operating System: Ubuntu 18.04 

Summary of the issue or suggestion: 
----------------------------------------------
When attempting to start Minecraft 1.7.2 with forge, it crashes immediately.

What should happen:
------------------------------
Minecraft opens with Forge mod-loader.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Create a Minecraft 1.7.2 instance
2. Go to "Edit instance"
3. Go to "Version"
4. Press "Install Forge"
5. Choose version 10.12.2.1121
5. Press "Launch"

Suspected cause:
---------------------------
 Forge modloader - instance works fine without Forge

Logs/Screenshots:
----------------------------
https://paste.ee/p/QjTsP

**Comment by ProjectSynchro** [comment]
This is a Forge bug, 
It was introduced with the release of Java 8u20, this was a widespread issue until it was patched.
Someone detailed what changes were made as well as mentioned a diff [here](https://github.com/ATLauncher/ATLauncher/issues/303#issuecomment-270939897)

So you're left with two options:
You can either run anything newer (1.7.10 or above)
Or you can add the coremod [LegacyJavaFixer,](https://github.com/LexManos/LegacyJavaFixer) which patches the class in forge at runtime. 

Not sure if there is a mirror hosted at MultiMC for this at this time.

**Comment by Forkk** [comment]
This issue is being closed for inactivity. If you're still having trouble with this, feel free to come by our [Discord server](https://discord.gg/multimc) and ask for help.
